File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/Features/CheckBoxPage.vb
Namespace CompanionKit.Controls.ListView.Features

	Public Class CheckBoxPage

		Public Sub New()

			' This call is required by the Visual WebGui UserControl Designer.
			InitializeComponent()

		End Sub

        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjGridCheck control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjGridCheck_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjGridCheck.CheckedChanged
            'Change ListView.GridLines property
            mobjListView.GridLines = mobjGridCheck.Checked
        End Sub

        ''' <summary>
        ''' Handles the CheckedChanged event of the mobjBoxesCheck control.
        ''' </summary>
        ''' <param name="sender">The source of the event.</param>
        ''' <param name="e">The <see cref="System.EventArgs"/> instance containing the event data.</param>
        Private Sub mobjBoxesCheck_CheckedChanged(sender As System.Object, e As System.EventArgs) Handles mobjBoxesCheck.CheckedChanged
            'Change ListView.CheckBoxes property
            mobjListView.CheckBoxes = mobjBoxesCheck.Checked
        End Sub
    End Class

End Namespace

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/ListView/Features/CheckBoxPage.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.ListView.Features
{
    public partial class CheckBoxPage : UserControl
    {
        public CheckBoxPage()
        {
            InitializeComponent();            
        }


        /// <summary>
        /// Handles the CheckedChanged event of the mobjGridCheck control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjGridCheck_CheckedChanged(object sender, EventArgs e)
        {
            //Change ListView.GridLines property
            mobjListView.GridLines = mobjGridCheck.Checked;
        }

        /// <summary>
        /// Handles the CheckedChanged event of the mobjBoxesCheck control.
        /// </summary>
        /// <param name="sender">The source of the event.</param>
        /// <param name="e">The <see cref="EventArgs"/> instance containing the event data.</param>
        private void mobjBoxesCheck_CheckedChanged(object sender, EventArgs e)
        {
            //Change ListView.CheckBoxes property
            mobjListView.CheckBoxes = mobjBoxesCheck.Checked;
        }
    }
}